components/lib/crash/src/main/java/mozilla/components/lib/crash/service/GleanCrashReporterService.kt

```diff
@@ -52,11 +51,6 @@ class GleanCrashReporterService(
                 return@run
             }
 
-            if (!Glean.isInitialized()) {
```

Comment:
This change is out of scope for this PR. Let's take it out and have it as a follow-up PR.